.gitignore

```diff
@@ -4,4 +4,7 @@ tags
 composer.lock
 vendor
 npm-debug.log
-.idea
\ No newline at end of file
+.idea
+
+## IDE
+.vscode
```

Comment:
You can move `.idea` down here as well - that's the container for PhpStorm/WebStorm.